pkg/common/CommonPackage.props

```diff
@@ -11,11 +11,13 @@
              Condition="'$(PlatformTarget)' == 'x64'">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
+	  <Link>%(Filename)%(Extension)</Link>
```

Comment:
(nit) can you fix the whitespace here? The file is currently using spaces, and you've inserted tabs.

Reply:
Fixed